package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@biconomy/mexa",
-  "version": "2.0.10",
+  "version": "2.0.12",
```

Comment:
2.0.12 is deployed currently. Change it to 2.0.13

Reply:
it shows here 2.0.13 is deployed
https://www.npmjs.com/package/@biconomy/mexa

will make it 2.0.14

Reply:
done